UI/MemStickScreen.h

```diff
@@ -30,19 +30,52 @@
 // Currently only useful for Android.
 class MemStickScreen : public UIDialogScreenWithBackground {
 public:
-	MemStickScreen() {}
-    virtual ~MemStickScreen();
+	MemStickScreen(bool initialSetup);
+	~MemStickScreen() {}
 
 	std::string tag() const override { return "game"; }
+	void CreateViews() override;
```

Comment:
Hm, what's the reason for this to be public?  Only RecreateViews() should get called right?

-[Unknown]

Reply:
No reason, fixing.